C/CombBLAS/build_tarballs.jl

```diff
@@ -0,0 +1,74 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message
+using BinaryBuilder, Pkg
+
+name = "CombBLAS"
+version = v"2.0.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource(
+        "https://github.com/PASSIONLab/CombBLAS.git",
+        "e1c7faad8d5a918d5671b794c04672f30c6bec29",
+    ),
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd ${WORKSPACE}/srcdir/CombBLAS/
+install_license LICENSE
+mkdir build
+cd build
+# Force gcc on all platforms
+cmake -DCMAKE_INSTALL_PREFIX=${prefix} \
+      -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN%.*}_gcc.cmake \
```

Comment:
Why forcing GCC everywhere?

Reply:
Maybe my comment is a bit too brute/lacking some context 😅 There have been [some unaddressed issues with clang](https://github.com/search?q=repo%3APASSIONLab%2FCombBLAS+clang&type=issues) + CombBLAS[ is built with GCC in mind](https://github.com/search?q=repo%3APASSIONLab%2FCombBLAS%20gcc&type=code)

Reply:
When you build with GCC also on BSD platforms you need to expand c++ string ABI also there.

Reply:
Do you mean `expand_cxxstring_abis`? If so, that is included. Also, CombBLAS does not build on FreeBSD, so that platform's removed. I just changed up the MPI implementation to match other packages so the build recipe should follow convention.

Reply:
Yes, but by default [`expand_cxxstring_abis`](https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.expand_cxxstring_abis-Tuple{AbstractPlatform}) doesn't expand the ABI for BSD platforms.

Reply:
`expand_cxxstring_abis(skip = Returns(false))`?